sonar-db/src/main/java/org/sonar/db/component/ResourceKeyUpdaterDao.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.db.component;
 
+import com.google.common.base.Preconditions;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.base.Preconditions'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)